engine/resource/src/dmsdk/resource/resource_doc.h

```diff
@@ -0,0 +1,260 @@
+/*# Resource
```

Comment:
To keep the resource.h more readable at-a-glance, I moved some of the documentation here.